.github/workflows/test.yml

```diff
@@ -311,16 +311,6 @@ jobs:
           - name: windows
             runner: windows-latest
     steps:
-      # On Windows, set autocrlf to input so that when the repo is cloned down
-      # the fixtures retain their line endings and don't get updated to CRLF.
-      # We want this because this repo also contains the fixtures for our test cases
-      # and these fixtures have files that need stable file hashes. If we let git update
-      # the line endings on checkout, the file hashes will change.
-      # https://www.git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#_core_autocrlf
-      - name: set crlf
-        if: matrix.os.name == 'windows'
-        shell: bash
-        run: git config --global core.autocrlf input
```

Comment:
Curious.. what about this is contributing to the hash for `cli#build` such that we need to find an alternative solution? And if this is the cause, would it not make sense to also add this step to the `build_turborepo` job?

It seems harder to ensure that every fixture has a .gitattributes

Reply:
We can do this, but we just need to make sure that every Windows job has this set. I was thinking it could be easier just to set the test fixtures directory up with `LF` eol to avoid this, but that seems to be just as sprawling.